_data/faq.yml

```diff
@@ -1,3 +1,131 @@
+- id: design
+  title:
+      en: Recent changes to the c:geo design
+      de: Kürzliche Design-Änderungen in c:geo
+  items:
+      - id: design-generic
+        en:
+            title: Why did you change the c:geo design?
+            content: |
+                Major parts of c:geo have been designed many years ago and were targeted to very old Android versions. Some of those designs have not even been touched ever since.
+                Over the last years much effort was needed to bugfix and patch this really old design to make it usable for new screens and features.
+                
+                Finally we ended up with a very unstructured design, which made c:geo look inconsistent or even broken at many places.
+                Additionally the old design did not allow us to make use of modern app design principles and features, which you know from many other Android apps.
+                
+                Therefore we finally decided to start from scratch and implement the so-called material design into c:geo. 
+                We tried our best to implement it in a way, that you will still recognize c:geo but have a more consistent and structured appearance.
+                
+                At some places this is still work in progress and we are unable to test it on all the different Android devices and versions.
+                Thus there might still be some minor bugs. Feel free to provide <a href="mailto:support@cgeo.org?subject=New design">feedback to our support</a> if you encounter problems or have improvement proposals.
+        de:
+            title: Warum wurde das c:geo Design geändert?
+            content: |
+                TBC     
+      - id: design-background
+        en:
+            title: Where is my background image?
+            content: |
+                The transparent background was a rather old feature and is no longer used in modern designs. Therefore this feature has been removed.
+                
+                We are planning to change the main menu as well in future versions and migrate to a bottom navigation with a dashboard. 
+                It might be possible, that such dashboard will bring back a similar customization option. We are still working on it.
+        de:
+            title: Wo ist mein Hintergrundbild?
+            content: |
+                TBC  
+      - id: design-font
+        en:
+            title: Why are all texts now bigger?
+            content: |
+                We do now follow the Android style guidelines for font sizes. We also had many users telling us, that the texts are rather small and hard to read compared to other apps on the same device.
+                
+                Give it a try for some days...its probably just a matter of getting used to the new look.
+                Besides that, you can now also adjust the font size in c:geo by adjusting your system settings accordingly as c:geo is now following that system setting.                
```

Comment:
This has not changed with recent UI overhaul, but does already work since a long time in our published releases.

Reply:
No, not for me on my Samsung for some reason. Only some elements got resized (e.g. description) but many other places kept unchanged.
Same experience was reported from Nokia users ever since (but now confirmed working). Dont ask me why.

Reply:
I removed the "now" from the sentence. That should fit.